app/assets/stylesheets/_animations.scss

```diff
@@ -0,0 +1,19 @@
+.scaleIn-animation {
```

Comment:
We should not have camelCase on CSS class names, what do you think about simply `scale-in`? Adding `animation` to the class name also doesn't add semantics.